### PR TITLE
autotest scene pre parallel steps output shouldn't show between parallel steps

### PR DESCRIPTION
**Pull request by kakj-go**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
autotest scene pre parallel steps output shouldn't show between parallel steps

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/render.go [hunk 1]
```diff
@@ -157,12 +157,12 @@ LABEL:
 		if sStep.ID == stepID {
 			break
 		}
-		steps = append(steps, sStep)
 		for _, pStep := range sStep.Children {
 			if pStep.ID == stepID {
 				break LABEL
 			}
 		}
+		steps = append(steps, sStep)
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/render.go:165]: can you explain what logic in it?

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/render.go (hunk 1 above)
Comment (kakj-go) [reply modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/render.go:165]: The old logic is to judge whether it is a parallel node first, and it should be a parallel node before attaching

**Review by sfwn — APPROVED** [review]

**Pull request merged**